doc/SeparatingVttabletMysql.md

```diff
@@ -33,7 +33,7 @@ to make sure the tablet type in the topology is correct before running
 vttablet. Usually, vttablet can be started with `-init_tablet_type replica`,
 even for a master (as `master` is not allowed), and will figure out the master
```

Comment:
i know you are planning to do the website changes after 11.0, so i'm not sure if you want to tackle these at the same time but there's a bunch of "master" in this doc

Reply:
Once we get through all the known tasks I plan to do another `grep` through the code base and fix anything left behind.
I'll add this to the task list. So far I've been grepping only `go` files.